src/main/java/randoop/main/GenTests.java

```diff
@@ -1074,13 +1154,15 @@ private void printSequenceExceptionError(AbstractGenerator explorer, SequenceExc
    *
    * @param visibility the visibility predicate
    * @param contracts the contract checks
-   * @param observerMap the map from types to observer methods
+   * @param omitMethodsPredicate the predicate for filtering out omit methods
+   * @param sideEffectFreeMethods the map from types to side-effect-free methods
```

Comment:
This name `sideEffectFreeMethods` is better than `sideEffectFreeMap` which is used elsewhere.